server/member.go

```diff
@@ -99,7 +99,7 @@ func (s *Server) registerMemberRoutes(g *echo.Group) {
 		memberFind := &api.MemberFind{
 			ID: &id,
 		}
-		member, err := s.store.FindMember(ctx, memberFind)
+		member, err := s.store.GetMember(ctx, memberFind)
```

Comment:
Then it'll split into `GetMemberByID` and `GetMemberByPrincipalID`, because there are currently 2 kinds of `MemberFind` usage.
Should we keep it or split it into these two functions above? WDYT @d-bytebase

Reply:
If there are not many occurrences, we can keep using "find" but the caller needs to assert the returned list size. Otherwise, we can have two separate methods, GetMemberByID and GetMemberByPrincipalID.